### DISCUSSION TITLE
Question: will these be signed for secure boot?

### DISCUSSION THREAD

**Discussion by xsrvmy** [NVIDIA/open-gpu-kernel-modules#220] [General]
With the open source gpu kernel modules, will we eventually be able to install nvidia drivers without having to go through the manual signing process on secure-boot enabled machines? Or will the process of doing the manual signing remain the same?

**Comment by YusufKhan-gamedev** [comment]
See https://askubuntu.com/questions/1023036/how-to-install-nvidia-driver-with-secure-boot-enabled

**Reply by sTiKyt** [reply]
That's not what he asked

**Comment by xsrvmy** [comment]
I know it can be self signed. I was just wondering that will be avoided in the future.

**Comment by Evernow** [comment]
That'll likely not be useful since distros will build this themselves, they'll have to sign it.
